themes/base.theme.bash

```diff
@@ -396,7 +396,7 @@ function hg_prompt_vars {
 
 function nvm_version_prompt {
 	local node
-	if declare -f -F nvm &> /dev/null; then
+	if _command_exists nvm; then
```

Comment:
Since this was specifically testing for being a function, you might want to use [`_is_function`](https://github.com/Bash-it/bash-it/blob/master/lib/helpers.bash#L167)